debug/vector/blanket.html

```diff
@@ -0,0 +1,71 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title>Leaflet debug page</title>
+
+	<meta name="viewport" content="initial-scale=1.0, user-scalable=no" />
+
+	<link rel="stylesheet" href="../../dist/leaflet.css" />
+
+	<link rel="stylesheet" href="../css/mobile.css" />
+</head>
+<body>
+	<div id="map"></div>
+
+	<script type="module">
+		// import {tileLayer, Map, LatLng, BlanketOverlay, Browser, Canvas, SVG, CircleMarker} from '../../src/Leaflet.js';
+		import {tileLayer, Map, LatLng, BlanketOverlay, Browser, Canvas, SVG, CircleMarker} from '../../dist/leaflet-src.esm.js';
+
+		const osmUrl = 'https://tile.openstreetmap.org/{z}/{x}/{y}.png',
+			osmAttrib = '&copy; <a href="https://www.openstreetmap.org/copyright">OpenStreetMap</a> contributors',
+			osm = tileLayer(osmUrl, {maxZoom: 18, attribution: osmAttrib});
+
+		const map = window.map = new Map('map', {

```

Comment:
do we want to keep `window.map`?

Reply:
Oh, that was meant for debugging (unlike non-module scripts, variables defined don't become globals and cannot be referenced in the console). Will delete.